shared/src/commonMain/kotlin/org/hyperskill/app/home/presentation/HomeFeature.kt

```diff
@@ -12,11 +13,13 @@ interface HomeFeature {
     data class State(
         val homeState: HomeState,
         val toolbarState: GamificationToolbarFeature.State,
+        val problemsLimitState: ProblemsLimitFeature.State,
         val topicsToDiscoverNextState: TopicsToDiscoverNextFeature.State
     ) {
         val isRefreshing: Boolean
             get() = homeState is HomeState.Content && homeState.isRefreshing ||
                 toolbarState is GamificationToolbarFeature.State.Content && toolbarState.isRefreshing ||
+                problemsLimitState is ProblemsLimitFeature.State.Content && problemsLimitState.isRefreshing ||
```

Comment:
It would be more useful to have an extension property `ProblemsLimitFeature.State.isRefreshing` as a shortcut for `problemsLimitState is ProblemsLimitFeature.State.Content && problemsLimitState.isRefreshing`.
If all embedded features would have that extension we can reduce amount of code for all parents `isRefreshing` properties.

Reply:
@vladkash @ivan-magda
What do you think?

Reply:
I think it's useful because we reuse this logic for screens with pull-to-refresh functionality (will use it for study plan screen)